Sender/BMS_sendertest.cpp

```diff
@@ -0,0 +1,65 @@
+/*--------------------------------------------------------------------------------
+---    \file       BMS_sendertest.cpp
+---   \author      Naveen Balachandar (RBEI/ECK3) 
+---
+---     \brief     unit testing file to test streaming BMS data
+------------------------------------------------------------------------------------*/
+#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
+
+#include "../test/catch.hpp"
+#include "BMS_Sender.h"
+
+TEST_CASE("Test case 1 : Console output sent status") 
+{
+    /*Inital conditions*/	
+    streamAlert_Status_s streamAlertTestStat = {NOT_SENT,FILE_ACCESS_FAILURE};
+    /*func call with stub data*/
+    streamAlertTestStat = sendDataToConsole(11.5, 0.7);
+ 
+    /*Validation success*/	  
+    REQUIRE(streamAlertTestStat.ConsoleSentStatus == SENT_TO_CONSOLE);
```

Comment:
The output format is the crucial functionality of the sender. Variations in that can throw the receiver out of gear. If future maintainers introduce an issue in the output format, it will be detected only during integration. That would take a lot more time than a simple unit-test.
Also, writing code to parse the output will give you a clue how complex the sender will need to be, for your output format.